From 1ca7720395caaf9a4e23db5521ea699cc6a60b49 Mon Sep 17 00:00:00 2001
From: Unknown <unkonwn@invalid>
Date: Sun, 17 May 2026 13:51:10 +0000
Subject: [PATCH 01/13] pinebook-pro: Add PREBOOT to start USB before boot menu

Ensure that USB keyboards are available when the boot menu appears.
---
 configs/pinebook-pro-rk3399_defconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/configs/pinebook-pro-rk3399_defconfig b/configs/pinebook-pro-rk3399_defconfig
index cbf95b4692f..af09eb18b13 100644
--- a/configs/pinebook-pro-rk3399_defconfig
+++ b/configs/pinebook-pro-rk3399_defconfig
@@ -22,6 +22,7 @@ CONFIG_PCI=y
 CONFIG_DEBUG_UART=y
 CONFIG_BOOTDELAY=3
 CONFIG_USE_PREBOOT=y
+CONFIG_PREBOOT="usb start"
 CONFIG_DEFAULT_FDT_FILE="rockchip/rk3399-pinebook-pro.dtb"
 CONFIG_DISPLAY_BOARDINFO_LATE=y
 CONFIG_SPL_MAX_SIZE=0x40000
-- 
2.54.0

